src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+import java.lang.*;
+import java.util.*;
+import java.io.File;
+
 public class Duke {
+    static String indent = "     ";
+    static String line = "    ____________________________________________________________";
+    private static final String FILEPATH = ""
```

Comment:
Good job making the constant variable in screaming snake case.

Reply:
I agree. Common Naming mistake. Good Job!